### PR TITLE
CRAFT-405- Engage default- Add support to Business Banking- public repo

### PR DESCRIPTION
**Pull request by zahraakbarib**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -94,8 +94,11 @@ jobs:
       - name: Build Engagements Data docker image
         shell: bash
         run: |
-          echo "ECR Docker build with tag ${NEW_VERSION}"
-          mvn clean package -Pretail-build -B docker:tag -Ddocker.image.tag=${NEW_VERSION} -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+          echo "ECR Docker build retail with tag ${NEW_VERSION}"
+          mvn clean package -Pbuild-retail -B docker:tag -Ddocker.image.tag=${NEW_VERSION} -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+
+          echo "ECR Docker build business with tag ${NEW_VERSION}"
+          mvn clean package -Pbuild-business -B docker:tag -Ddocker.image.tag=${NEW_VERSION} -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```
Comment (eacarvalho) [inline .github/workflows/build.yml:101]: Please also update the "Use maven-docker-plugin to build and push docker image to ECR" part with what we have on ModelBank

File: README.md [hunk 2]
```diff
@@ -44,22 +44,56 @@ This repository has the following structure:
...
+        │   ├── templates                                       # The folder to store templates
+        │   │   ├── {template-name}
+        │   │   │   ├── content-schema.json                     # A structure of an engagement template based on a given template
+        │   │   │   └── template.json                           # A template definition
+        │   │   └── ...
+        │   └── placeholders                                    # The folder to store placeholder group
+        │       ├── {placeholder-group-name}
+        │       │   └── placeholders.json                       # A placeholder group configuration
+        │       └── ...
+        └── repositories                                        # The folder with a repository definition.
```
Comment (eacarvalho) [inline README.md:89]: Please check again the repositories configuration folder and the create-collections.sh file with what we have on ModelBank

**Review by eacarvalho — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 3]
```diff
@@ -7,12 +7,12 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.backbase.modelbank</groupId>
     <artifactId>engagements-data</artifactId>
-    <version>0.10.2</version>
+    <version>0.11.0</version>
```
Comment (eacarvalho) [inline pom.xml:10]: I'm quite sure you can keep the `0.10.2` version and the pipeline will bump automatically, no?

**Review by eacarvalho — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (zahraakbarib) [reply pom.xml:10]: ok. I will keep the 0.10.2

**Review by eacarvalho — APPROVED** [review]

**Pull request merged**